### PR TITLE
DEV-894: Feature list

### PR DESCRIPTION
**Pull request by Lax125**
# Description

Adds feature list to doco.

# Associated tasks

No blocking nor blocked tasks.

# Checklist

If any of these are not applicable, strikethrough the line `~like this~`. **Do not delete it!**. Let the reviewer decide if you should have done it.

### Code
- [x] I have performed a self review of my own code (including checking issues raised when creating the PR).
~- [ ] I have added/updated unit tests for these changes, and if not I have explained why they are not necessary.~ (No tests needed for doco change)
~- [ ] I have commented my code in any hard-to-understand or hacky areas.~
- [x] I have handled all new warnings generated by the compiler or IDE.
- [x] I have rebased onto the target branch (usually main).
      
### Documentation
- [x] I have updated the changelog.
- [x] I have updated any documentation required for these changes.

# Breaking Changes
- [x] I have considered if this is a breaking change and will communicate it with other team members if so.

No breaking changes.

# Screenshots

| Before | After |
| --- | --- |
| (No feature list) | image |

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [DEV-894 Write capability matrix for SDKs (in Python SDK)](https://app.clickup.com/t/6929263/DEV-894)

File: docs/sidebars.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
             items: ["venv-setup", "intellij-setup"],
             label: "Setup Guide"
         },
+        "sdk-overview",
```
Comment (Lax125) [inline docs/sidebars.js:16]: Should this go before the setup guide?

Comment (Lax125) [reply docs/sidebars.js]: Moved to before setup guide, since users will want to see an overview of what the product is about first.

**Comment by charlta** [comment]
as per the comment on the JVM version, we now need to update the python SDK if we add a feature to the JVM version. I think these should be isolated to prevent doco only updates to each repo.

**Comment by Lax125** [comment]
Will change to feature list.

**Review by zarakay — APPROVED** [review]

**Pull request merged**